v0.12/debian/entrypoint.sh

```diff
@@ -21,8 +9,20 @@ if [ -r $DEFAULT ]; then
     set +o allexport
 fi
 
-# chown home and data folder
-chown -R fluent /home/fluent
-chown -R fluent /fluentd
+# If the user has supplied only arguments append them to `fluentd` command
+if [ "${1:0:1}" = '-' ]; then
+    set -- fluentd "$@"
+fi
+
+# If user does not supply config file or plugins, use the default
+if [ "$1" == "fluentd" ]; then
+    if ! echo $@ | grep '\-c' ; then
+       set -- "$@" -c /fluentd/etc/fluent.conf
+    fi
+
+    if ! echo $@ | grep '\-p' ; then
+       set -- "$@" -p /fluentd/plugins
+    fi
+fi
 
 exec gosu fluent "$@"
```

Comment:
Would it be worthwhile to update this for allowing arbitrary `--user` values? ie something like:

```bash
if [ "$(id -u)" = '0' ]; then
	exec gosu fluent "$@"
else
	exec "$@"
fi
```

Reply:
Not sure this is required for this project.

Best ask @repeatedly.

Reply:
Is it important for official images?
We can add this change after become official, right?

Reply:
Judging by the language @tianon uses here, I would say that it is not a blocker for Official Images.

Thus adding this change as a subsequent PR sounds reasonable.